src/rewriter/mkrewrites.py

```diff
@@ -134,7 +88,11 @@ def gen_mk_node(defns, expr):
         return expr.name
     elif isinstance(expr, App):
         args = ",".join(gen_mk_node(defns, child) for child in expr.children)
-        return f'nm->mkNode({gen_kind(expr.op)}, {{ {args} }})'
+        if (expr.op == Op.EXTRACT or expr.op == Op.REPEAT or expr.op == Op.ZERO_EXTEND or expr.op == Op.SIGN_EXTEND or expr.op == Op.ROTATE_LEFT or expr.op == Op.ROTATE_RIGHT or expr.op == Op.INT_TO_BV):
```

Comment:
```suggestion
        if expr.op == Op.EXTRACT or expr.op == Op.REPEAT or expr.op == Op.ZERO_EXTEND or expr.op == Op.SIGN_EXTEND or expr.op == Op.ROTATE_LEFT or expr.op == Op.ROTATE_RIGHT or expr.op == Op.INT_TO_BV:
```

Make sure to stay within 80 chars per line.

Reply:
fixed